spec/models/reporting/processed_report_spec.rb

```diff
@@ -15,11 +15,11 @@
     end
 
     it 'includes the row headers' do
-      expect(rows.map(&:first).map(&:content)).to eq(['Today', 'Yesterday', 'Day before yesterday'])
+      expect(rows.map { |x| x.first.content }).to eq(['Today', 'Yesterday', 'Day before yesterday'])
```

Comment:
Such a silly tiny thing, but would you be up for changing x to r (for row)?

Reply:
Or just "row" 🤔